external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -781,7 +796,19 @@ impl UseDef {
             fhash: def_fhash,
             start: def_start,
         };
-        let col_end = use_start.character + use_name.len() as u32;
+        // Normally, we compute the length of the identifier as the length
```

Comment:
The actual new feature implementation would be very simple if not for aliasing and this part is where it's taken care of.